scenes/game/local_game.gd

```diff
@@ -4006,14 +4006,14 @@ func do_card_from_hand_to_gauge(performing_player : Player, card_ids : Array) ->
 		if not performing_player.is_card_in_hand(card_id):
 			printlog("ERROR: Tried to do_card_from_hand_to_gauge with card not in hand.")
 			return false
-			
+	
 	var events = []
 	if card_ids.size() > 0:
 		var card_names = card_db.get_card_name(card_ids[0])
 		for i in range(1, card_ids.size()):
 			card_names += ", " + card_db.get_card_name(card_ids[i])
 		var destination_string = ""
-
+		
```

Comment:
I like to use VScode with the option to eliminate extra whitespace when you save so you don't get random stuff like this.
It doesn't matter though, don't worry about removing these.

Reply:
I've just been using the built-in godot editor; i'll try to see if there's anything similar in the settings